staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -459,11 +500,11 @@ func (f *featureGate) SetFromMap(m map[string]bool) error {
 	}
 	f.enabledRaw.Store(enabledRaw)
 
-	errs := f.unsafeSetFromMap(enabled, enabledRaw, f.EmulationVersion(), f.MinCompatibilityVersion())
+	errs := f.unsafeSetFromMap(logger, enabled, enabledRaw, f.EmulationVersion(), f.MinCompatibilityVersion())
 	if len(errs) == 0 {
 		// Persist changes
 		f.enabled.Store(enabled)
-		klog.V(1).Infof("feature gates: %v", f.enabled)
```

Comment:
`f.enabled` is `atomic.Value` (a struct which must not be copied!), not the map.